plutus-core/plutus-core/test/type-errors/errorKind.plc.golden

```diff
@@ -1,2 +1,3 @@
-E015: Kind mismatch at line 2, column 4 in type '(lam
-  a (type) a)'. Expected kind '(type)' , found kind '(fun (type) (type))'
\ No newline at end of file
+E015: Kind mismatch at line 2, column 4 in type '(lam a
```

Comment:
This didn't actually work

Reply:
It's now worse than previously IMO :laughing: 

Maybe let's just revert this "reversion" and worry about the hard breaks later.